Lateral Movement/ServiceAccountsPerformingRemotePS.txt

```diff
@@ -0,0 +1,55 @@
+// Service Accounts Performing Remote PowerShell
+// Author: miflower
+// The purpose behind this detection is for finding service accounts that are performing remote powershell sessions
+// There are two phases to the detection: Identify service accounts, Find remote PS cmdlets being ran by these accounts
+// To accomplish this, we utilize LogonEvents and MiscEvents to find cmdlets ran that meet the criteria
+// One of the main advantages of this method is that only requires server telemetry, and not the attacking client
+// The first phase relies on the LogonEvents to determine whether an account is a service account or not, consider the following accounts with logons:
+// random_user has LogonEvents with type 2, 3, 7, 10, 11 & 13
+// random_service_account 'should' only have LogonEvents with type 3,4 or 5
+//
+let InteractiveTypes = pack_array(                                  // Declare Interactive logon type names
+    'Interactive',
+    'CachedInteractive',
+    'Unlock',
+    'RemoteInteractive',
+    'CachedRemoteInteractive',
+    'CachedUnlock'
+);
+let WhitelistedCmdlets = pack_array(                                // List of whitelisted commands that don't provide a lot of value
+    'prompt',
+    'Out-Default',
+    'out-lineoutput',
+    'format-default',
+    'Set-StrictMode',
+    'TabExpansion2'
+);
+let WhitelistedAccounts = pack_array('FakeWhitelistedAccount');     // List of accounts that are known to perform this activity in the environment and can be ignored
+LogonEvents                                                         // Get all logon events...
+| where AccountName !in (WhitelistedAccounts)                       // ...where it is not a whitelisted account...
```

Comment:
Change it to !in~
We lowercase all account names, IIRC, so could compare case sensitive - but still this change would probably save some trouble to some users....

Reply:
Added, will be in next PR.